javascript/implementation/GinkInstance.ts

```diff
@@ -400,19 +402,25 @@ export class GinkInstance {
      * @param resolveOnOpen if true, resolve when the connection is established, otherwise wait for greeting
      * @returns a promise to the peer
      */
-    public async connectTo(target: string, onClose: CallBack = noOp, resolveOnOpen?: boolean): Promise<Peer> {
+    public async connectTo(
+        target: string,
+        options?: {
+            onClose?: CallBack,
+            resolveOnOpen?: boolean,
+            authToken?: string;
+        }): Promise<Peer> {
         //TODO(https://github.com/google/gink/issues/69): have the default be to wait for databases to sync
+        const onClose: CallBack = (options && options.onClose) ? options.onClose : noOp;
+        const resolveOnOpen: boolean = (options && options.resolveOnOpen) ? options.resolveOnOpen : false;
+        const authToken: string = (options && options.authToken) ? options.authToken : undefined;
+        if (authToken && !authToken.toLowerCase().startsWith("token ")) throw new Error("auth token needs to start with 'token '");
```

Comment:
If we're going to take a parameter named `authToken` we should probably just accept a straight token, and not require users to prepend the `token` bit.